torch_xla/csrc/ops/moving_average.h

```diff
@@ -6,24 +6,20 @@ namespace torch_xla {
 namespace ir {
 namespace ops {
 
-class BatchNormForward : public Node {
+class MovingAverage : public Node {
```

Comment:
This isn't really a moving average, it's just a linear interpolation. Maybe rename it?

Reply:
This is used as moving average from BN but I agree, we can have to low level op be named something more generic.